sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/text/TextSuite.scala

```diff
@@ -65,6 +65,14 @@ class TextSuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  test("support for partitioned reading") {
+    val df = spark.read.format("text").load(testDir)
```

Comment:
This semantic makes sense to me.  But can we generate the test files dynamically instead of checking them in? (thats what we do in all the other tests)